nerdlandbot/commands/recipe.py

```diff
@@ -0,0 +1,153 @@
+import typing
+import discord
+from discord.ext import commands
+import gspread
+import os
+import asyncio
+from datetime import datetime
+
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.constants import NOTIFY_EMBED_COLOR
```

Comment:
non blocking comment: how about a seperate color here? since it's not notify :)

Reply:
I actually never realised it was for the notify.. For some reason I always read it as "STANDARD_NOTIFY_COLOR". Since I've also used this in other places, maybe I can do so when this gets approved and there has been a decision on a guideline for using embeds?